.github/workflows/ci.yml

```diff
@@ -3,25 +3,17 @@ name: Ubuntu CI
 on: [push, pull_request]
 
 jobs:
-  focal-ci:
+  jammy-ci:
     runs-on: ubuntu-latest
-    name: Ubuntu Focal CI
+    name: Ubuntu Jammy CI
     steps:
       - name: Checkout
         uses: actions/checkout@v3
       - name: Compile and test
         id: ci
-        uses: gazebo-tooling/action-gz-ci@focal
+        uses: gazebo-tooling/action-gz-ci@jammy
         with:
           codecov-enabled: true
           cppcheck-enabled: true
           cpplint-enabled: true
-  jammy-ci:
-    runs-on: ubuntu-latest
-    name: Ubuntu Jammy CI
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-      - name: Compile and test
-        id: ci
-        uses: gazebo-tooling/action-gz-ci@jammy
+          doxygen-enabled: true
```

Comment:
this picked up a few doxygen warnings. Maybe we should fix those before enabling?

Reply:
Fixed warnings in https://github.com/gazebosim/gz-sensors/pull/364/commits/ccec18dcee45103fede09a9ed598b49179819b77